backend-rust/migrations/0001_initialize.up.sql

```diff
@@ -309,11 +310,47 @@ CREATE TABLE contracts(
     init_transaction_index
         BIGINT
         NOT NULL,
+    -- The version of the smart contract.
+    version
+        INT
+        NOT NULL,
 
     -- Make the contract index and subindex the primary key.
     PRIMARY KEY (index, sub_index)
 );
 
+-- Every successful event associated to a contract.
+CREATE TABLE contract_events (
+    -- An index/id for this event (row number).
+    index
+        BIGINT GENERATED ALWAYS AS IDENTITY
```

Comment:
Didn't we discuss that it would be nice if these indexes were autogenerated and sequentially added (without skipping numbers)?

Reply:
My understanding from [that discussion](https://github.com/Concordium/concordium-scan/pull/268#discussion_r1808299392) was that we concluded that using plain integers was fine since the insertion is sequential. But it's not hugely important.